apps/frontend/translations/en.json

```diff
@@ -221,7 +221,7 @@
       }
     }
   },
-  "provinces": {
```

Comment:
why the singularization?

Reply:
it is more logical and I used it in the table header